### DISCUSSION TITLE
Project Management

### DISCUSSION THREAD

**Discussion by ayushozha** [dailydotdev/daily#471] [Feature Requests]
I would like to request a tag for project management. Please have this tag include all the subcategories of project management, including tools sections that have articles and blogs for different project management tools.

**Comment by idoshamun** [comment]
Hi @Ayush10,

Our focus at the moment is software engineering with that said please feel free to submit new sources that you think might fit our community. Here's a guide on how to submit new sources, make sure to read our guidelines:
https://docs.daily.dev/docs/for-content-creators/suggest-new-source#how-to-submit-a-new-source-request
